docs/docs/aws.md

```diff
@@ -425,12 +426,7 @@ As introduced in the previous sections, `S3FileIO` adopts the latest AWS clients
 This means for any table manifests containing `s3a://` or `s3n://` file paths, `S3FileIO` is still able to read them.
 This feature allows people to easily switch from S3A to `S3FileIO`.
 
-If for any reason you have to use S3A, here are the instructions:
-
-1. To store data using S3A, specify the `warehouse` catalog property to be an S3A path, e.g. `s3a://my-bucket/my-warehouse` 
-2. For `HiveCatalog`, to also store metadata using S3A, specify the Hadoop config property `hive.metastore.warehouse.dir` to be an S3A path.
-3. Add [hadoop-aws](https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-aws) as a runtime dependency of your compute engine.
-4. Configure AWS settings based on [hadoop-aws documentation](https://hadoop.apache.org/docs/current/hadoop-aws/tools/hadoop-aws/index.html) (make sure you check the version, S3A configuration varies a lot based on the version you use).   
+If you do wish to use S3A, see the "Using the Hadoop S3A Connector" section below.
```

Comment:
```suggestion
If you do wish to use S3A, see the "Using Hadoop S3A via HadoopFileIO" section below.
```
(see heading comment below)

I think you might be able to link to that heading to help with navigation.